### PR TITLE
Fix #1387: Update the styled-components peerDependency 

### LINKED ISSUE #1387: Why is there discrepancy with peerDependencies?
We are using redoc on my team but I have noticed that when I do an npm-install with just a dependency of redoc-cli, I get a warning of 
```npm WARN redoc@2.0.0-rc.40 requires a peer of styled-components@^4.1.1 but none is installed. You must install peer dependencies yourself.```

But if I look at the dependencies for both redoc and redoc-cli, they both include a dependency for `"styled-components": "^5.1.1"` - so if I add a dependency for `styled-components@^4.1.1` I get a warning saying that `5.1` is needed but then when I add the `5.1` version then I get the warning above.

With these devDependencies I get the warning above:
```
  "devDependencies": {
    "mobx": "^4.2.0",
    "styled-components": "~5.1.1",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "redoc": "^2.0.0-rc.40"
  }
```

And then changing it to the `4.1.1` version:
```
  "devDependencies": {
    "mobx": "^4.2.0",
    "styled-components": "^4.1.1",
    "react": "^16.13.1",
    "react-dom": "^16.13.1",
    "redoc": "^2.0.0-rc.40"
  }
```

I get this warning:
```
npm WARN @redocly/react-dropdown-aria@2.0.11 requires a peer of styled-components@^5.1.1 but none is installed. You must install peer dependencies yourself.
```

### PR DESCRIPTION
**Pull request by tomgreen98**
In package.json we adjusted the styled-component peerDependecy from 4.11 to 5.1.1 to match the version listed in devDependency.

### CODE REVIEW

**Comment by tomgreen98** [comment]
This should resolve issue #1387

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
     "mobx": "^4.2.0 || ^5.0.0",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
-    "styled-components": "^4.1.1"
+    "styled-components": "^5.1.1"
```
Comment (RomanHotsiy) [inline package.json:135]:
```suggestion
    "styled-components": "^4.1.1 || ^5.1.1"
```

**Review by RomanHotsiy — APPROVED** [review]
Thanks for the PR!

**Comment by RomanHotsiy** [comment]
@tomgreen98 I am going cut a new release on Friday.

**Pull request merged**